symfony/framework-bundle/4.2/src/Kernel.php

```diff
@@ -25,6 +25,11 @@ public function registerBundles(): iterable
         }
     }
 
+    public function getProjectDir(): string
+    {
+        return dirname(__DIR__);
```

Comment:
`\dirname(__DIR__);` (dirname is part if the short list, see https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3048)

Reply:
Updated.